storybook/qmlTests/tests/tst_StatusButton.qml

```diff
@@ -0,0 +1,259 @@
+import QtQuick 2.15
+import QtTest 1.15
+
+import StatusQ.Controls 0.1
+
+import Models 1.0
+
+Item {
+    id: root
+    width: 600
+    height: 400
+
+    Component {
+        id: componentUnderTest
+        StatusButton {
```

Comment:
Not for now, but after short inspection I think that it would be enough and simpler to have just StatusButton with `flat` as an option instead separate `StatusButton` and `StatusFlatButton` duplicating logic of color selection (only primary is different).

Reply:
I agree yeah; also worth noting that we still have the "other" alien buttons `StatusRoundButton` and `StatusFlatRoundButton` which are still based on `Rectangle` instead of `StatusBaseButton` :/